web-client/src/presenter/sequences/editUnsignedDraftDocumentSequence.ts

```diff
@@ -17,7 +16,6 @@ export const editUnsignedDraftDocumentSequence = [
         isStatusReportOrder: [
           setEditStatusReportOrderFormAction,
           navigateToPathAction,
-          statusReportOrderPdfPreviewSequence,
```

Comment:
Thanks to https://github.com/ustaxcourt/ef-cms/pull/5254, I discovered this is a redundant call.

Reply:
As long as you tell me you have high confidence that our test suite covers the functionality this would impact on the slim chance something did go wrong with this, then I think it's reasonable 😄

Reply:
In addition to having tested manually, I've just double-checked that `edit-status-report-order.cy.ts` does check this (and fails successfully when I modify things further). Also, I've walked through the logic just to make sure I'm not lying to myself and taking advantage of your trust 🤣 

We call `statusReportOrderPdfPreviewSequence.ts` in three places:

1. `navigateToEditOrderSequence.ts` -- to show an existing order pdf if the user is editing
2. `gotoStatusReportOrderSequence.ts` -- to show an existing order pdf if the user is editing
3. `StatusReportOrder.tsx` -- to show the pdf when the user clicks "Preview"

In `navigateToEditOrderSequence.ts`, in the case of a status report order, we navigate to one of two "status-report-order-edit" routes, which both call `gotoStatusReportOrderSequence.ts`. However, in these edit cases, `gotoStatusReportOrderSequence.ts` itself calls `statusReportOrderPdfPreviewSequence.ts`. So any call to `statusReportOrderPdfPreviewSequence.ts` in `navigateToEditOrderSequence.ts` will always precede the call to `statusReportOrderPdfPreviewSequence.ts` in `gotoStatusReportOrderSequence.ts` and can thus be eliminated.